rpi/page.py

```diff
@@ -113,12 +113,12 @@ def __init__(self, main_view, *args, **kwargs):
 		self.result_string = tk.StringVar(value="")
 		self.result_message = tk.Label(self, textvariable=self.result_string, wraplength=500, justify="center", bg="#fff", font="helvetica 14")
 		
-		self.register_user_btn = tk.Button(self, text="Register", height=1, width=8, command=lambda: attempt_register(self, self.username_input.get(), self.password_input.get()))
+		self.register_user_btn = tk.Button(self, text="Register", height=1, width=8, command=lambda: attempt_register(self, self.username_input.get(), self.password_input.get(), self.fingerprintId))
 		
 		self.enrollment_result_string = tk.StringVar(value="")
 		self.enrollment_result_message = tk.Label(self, textvariable=self.enrollment_result_string, wraplength=500, justify="center", bg="#fff", font="helvetica 14")
 
-		self.fingerprint_enrollment_btn = tk.Button(self, text="Fingerprint Enrollment", height=1, width=8, command=lambda: enroll_fingerprint(self))
+		self.fingerprint_enrollment_btn = tk.Button(self, text="Enroll Fingerprint", height=1, width=16, command=lambda: enroll_fingerprint(self))
```

Comment:
Doesn't help much. I would be very careful with these terminologies as it should be as clear as possible. I like Fingerprint Enrollment a bit more